scripts/ci-test.sh

```diff
@@ -42,7 +49,12 @@ elif [[ -f "pyproject.toml" ]]; then
   poetry config virtualenvs.create true
   poetry config virtualenvs.in-project true
   (( SKIP_INSTALL == 0 )) && poetry install --no-interaction --no-ansi
-  poetry run pytest
+
+  if [[ "$source_folder" == *"/dbt" ]]; then
+    .venv/bin/dbt deps --project-dir "$dbt_project_folder" --profiles-dir "$dbt_project_folder"
+  fi
+
+  ENV=$ENV poetry run pytest
```

Comment:
@jalmari-sor how this is supposed to work?

Reply:
This file is called from buildspec.yml like this https://github.com/ringier-data/rcplus-alloy-star-tracking-data/blob/3f69a1162a0ad700397910813489068d2c175333/buildspec.yml#L22

In case we are testing a dbt project, we have to run dbt deps. 

Then the actual dbt test is executed as a pytest function that is here https://github.com/ringier-data/rcplus-alloy-star-tracking-data/blob/3f69a1162a0ad700397910813489068d2c175333/app/dbt/tests/test_execute.py#L19

This way we also dont have to do any extra definitions for the test script name here, or follow a convention for the naming and path